docs/faq.asciidoc

```diff
@@ -0,0 +1,51 @@
+[[faq]]
+== Frequently Asked Questions
+
+[float]
+[[faq-how-does-it-work]]
+=== How does the agent work?
+The agent auto-instruments known frameworks and libraries and records interesting events,
+like HTTP requests and DB queries.
+
+That works by leveraging the capability of the JVM to instrument the bytecode of classes so that for the supported technologies,
+there are no code changes required.
+The agent automatically injects small pieces of code before and after interesting events to measure their duration and metadata,
+like the DB statement,
+as well as HTTP related information,
+like the URL, parameters, and headers.
+For example, if the agent detects that a class extending `javax.servlet.HttpServlet` is loaded,
+it injects monitoring code before and after the servlet invocation.
+
+[float]
+[[faq-installation]]
+=== How do I install the agent?
```

Comment:
Do you think this question is really necessary in the FAQ? The getting started guide has the same information, and I don't think anyone would go to an FAQ before the getting started page when looking to install the product

Reply:
@makwarth what do you think?

Reply:
Agree